### PR TITLE
[ANCHOR-836] Add configuration `event.queue.kafka.ssl_cert_verify` 

### PR DESCRIPTION
**Pull request by lijamie98**
### Description

- Add configuration `event.queue.kafka.ssl_cert_verify` 

### Context

This flag is for dev and test environment where a valid CA may not be present. 

### Testing

- `./gradlew test`

### Documentation

N/A

### Known limitations

N/A

### CODE REVIEW

File: platform/src/main/resources/config/anchor-config-default-values.yaml [hunk 1]
```diff
@@ -702,6 +702,10 @@ events:
       # `sasl_mechanism` can be one of the following supported mechanisms:
       #   `PLAIN`: PLAIN SASL mechanism
       sasl_mechanism: PLAIN
+      # The configuration to enable SSL certificate verification. If true, the SSL certificate will be verified.
+      # For dev and test environments, it can be used to disable SSL certificate verification.
+      # For production environments, it is strongly recommended to set to true to avoid man-in-the-middle attach
```
Comment (philipliu) [inline platform/src/main/resources/config/anchor-config-default-values.yaml:707]: typo: attach should be attack

File: service-runner/src/main/resources/version-info.properties [hunk 2]
```diff
@@ -1 +1 @@
-version=2.10.2
\ No newline at end of file
+version=2.10.2R
```
Comment (philipliu) [inline service-runner/src/main/resources/version-info.properties:1]: What is the `R`?

Comment (lijamie98) [reply service-runner/src/main/resources/version-info.properties]: Thanks for catching the accidental key stroke.

File: platform/src/main/resources/config/anchor-config-default-values.yaml (hunk 1 above)
Comment (lijamie98) [reply platform/src/main/resources/config/anchor-config-default-values.yaml]: Thanks. Fixed.

**Review by philipliu — APPROVED** [review]

**Review by JiahuiWho — APPROVED** [review]

**Pull request merged**